

```diff
@@ -0,0 +1,19 @@
+update to home page image paths
+
+Signed-off-by: Cecilia Tran <cecilia.tran@dreamworks.com>
+
+# Please enter the commit message for your changes. Lines starting
```

Comment:
This file looks like it was committed accidentally. It's filename is all whitespace, and seems to be a commit message?

Reply:
To be clear, this is harmless and we could remove it later. No need to block landing this PR.